src/world_new/Robot.cpp

```diff
@@ -24,9 +24,6 @@ Robot::Robot(std::unordered_map<uint8_t, proto::RobotFeedback> &feedback, const
     if (id < 16) {
         workingDribbler = ai::Constants::ROBOT_HAS_WORKING_DRIBBLER(id);
         workingBallSensor = ai::Constants::ROBOT_HAS_WORKING_BALL_SENSOR(id);
-    } else {
-        std::cerr << "Warning: Creating robot with id = " << id << std::endl;
-        assert(false);
```

Comment:
I think it was a nice failsafe because it was thought that 'dummy' robot object never would be made. 

maybe it would be nicer to have a dummyRobot class extend on Robot, but i'd do so later.